packaging/suse/portus.spec.in

```diff
@@ -105,7 +105,7 @@ This package has been built with commit __COMMIT__ from branch __BRANCH__ on
 date __DATE__
 
 %prep
-%setup -q -n Portus-%{branch}
+%setup -q -n Portus-%{version}
```

Comment:
I don't think this is what we want. Imagine the packaging for `master`, or when we package commits which have not been tagged under any release yet.

Reply:
You are right! I had in mind only the branches.